### PR TITLE
feat: Added a CONTRIBUTING.md file.

### PR DESCRIPTION
**Pull request by ckaran**
This isn't complete yet as it is only partially adapted from
https://gist.github.com/PurpleBooth/b24679402957c63ec426 and
https://www.contributor-covenant.org/.  Needs another read-through
to make sure everything is in place.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+to make via issue, email, or any other method with the owners of this repository
+before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions
+with the project.
+
+## Intellectual Property Rights
+
+All contributions are only accepted under the license in the accompanying
+LICENSE.md file.  This means that when you make a contribution, you are agreeing
```
Comment (slightknack) [inline CONTRIBUTING.md:13]: I believe that the license file is just named `LICENSE` not `LICENSE.md`.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+to the terms in the LICENSE.md file, and are certifying that you have the
+necessary rights to make the contribution under the terms of that license.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the
```
Comment (slightknack) [inline CONTRIBUTING.md:19]: We might need to make this more specific to the project, as for now it's just a collection of markdown documents.

**Review by slightknack — COMMENTED** [review]
I left a few quick comments, but it otherwise looks good. Thanks for opening the first PR! 🎉

**Comment by slightknack** [comment]
For consistency, I've updated `LICENSE` to `LICENSE.md`. No further action is needed for now, so I'm merging it as it stands.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ckaran) [reply CONTRIBUTING.md:13]: Whoops!  I haven't checked, did you make the change?

File: CONTRIBUTING.md (hunk 2 above)
Comment (ckaran) [reply CONTRIBUTING.md:19]: Absolutely true.  I wanted to get this to you ASAP before you had a bunch of other contributors and had to add it in later.  Update it as you see fit.

**Pull request merged**